modules/web/src/app/project-overview/members-overview/template.html

```diff
@@ -61,7 +61,7 @@
       <button mat-flat-button
               (click)="membersAndGroupsNavigate()">
```

Comment:
We should also rename this method
```suggestion
              (click)="navigateToMembers()">
```